chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -40,11 +40,14 @@ use common::{
 use utils::ensure;
 use utxo::{
     BlockRewardUndo, BlockUndo, ConsumedUtxoCache, FlushableUtxoView, TxUndo, UtxosCache,
-    UtxosDBMut,
+    UtxosDBMut, UtxosView,
 };
 
 use self::error::ConnectTransactionError;
 
+pub struct Fee(pub Amount);
+pub struct Subsidy(pub Amount);
```

Comment:
It would be nice to add some doc-comments. The `Fee` type is pretty self explanatory, `Subsidy` is a little less.

Reply:
These types are pure syntactic sugar to not misuse Amount in arguments. Subsidy is as common term as Fee (meaning amount of new coins minted by new block). So I think comment is redundant here